storage/data-plane/pom.xml

```diff
@@ -3,49 +3,32 @@
  Licensed under the MIT License. See License.txt in the project root for
  license information.
 -->
-<!--
- Copyright (c) Microsoft Corporation. All rights reserved.
- Licensed under the MIT License. See License.txt in the project root for
- license information.
--->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.azure</groupId>
+        <artifactId>azure-client-sdk-parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
```

Comment:
Wes updated the version everywhere to be 1.0.0, so please update this also.

Reply:
updated